src/sentry/rules/actions/sentry_apps/notify_event.py

```diff
@@ -62,6 +62,23 @@ def _get_setting_value(self, field_name: str) -> str | None:
             None,
         )
 
+    def _get_sentry_app_installation_uuid(self) -> str | None:
+        sentry_app_installation_uuid = self.data.get("sentryAppInstallationUuid")
+        if not sentry_app_installation_uuid:
+            raise ValidationError("Missing attribute 'sentryAppInstallationUuid'")
+        return sentry_app_installation_uuid
+
+    def _get_alert_rule_component(self, sentry_app_id: int, sentry_app_name):
```

Comment:
Getting the alert rule component was moved into a separate function to decrease redundant code (used in existing function `self_validate` and new function `render_label`)